java/src/main/java/com/google/cloud/dataproc/templates/bigquery/README.md

```diff
@@ -34,4 +34,46 @@ bin/start.sh \
 --templateProperty bigquery.gcs.output.format=csv \
 --templateProperty bigquery.gcs.output.location=gs://output/csv \
 --templateProperty bigquery.gcs.output.mode=Overwrite
+```
+
+
+## BigQuery To JDBC
+
+General Execution:
+
+```
+export GCP_PROJECT=<gcp-project-id>
+export REGION=<region>
+export SUBNET=<subnet> 
+export GCS_STAGING_LOCATION=<gcs-staging-bucket-folder>
+export HISTORY_SERVER_CLUSTER=<history-server>
+
+bin/start.sh \
+-- --template BIGQUERYTOJDBC \
+--templateProperty project.id=<gcp-project-id> \
+--templateProperty bigquery.jdbc.input.table=<projectId:datasetId.tableName> \
```

Comment:
Ideally connector is accepting query as well table name. If we can provide one example with query then it would be great. Example if table has millions of row and anyone wants to query and load only subset then user might pass query instead of table name.

Reply:
Changes done. Please review it once.